src/devnet/external-server.ts

```diff
@@ -44,18 +45,20 @@ export async function getFreePort(): Promise<string> {
     );
 }
 
-export abstract class ExternalServer {
+export abstract class ExternalServer extends IntegratedDevnetLogger {
```

Comment:
I think this inheritance is wrong (not all ExternalServers are IntegratedDevnetLoggers). By doing this, all classes inheriting from ExternalServer will also start logging, right? Including StarknetVenvProxy.

In fact, I'm not even sure if we want to achieve this by making ExternalServer a child class of a logger class.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/179/commits/cfd12c44d12d31196d3a51293bdf70b855faa9fe